### ISSUE TITLE
Error when trying build project with --prod 

### ISSUE THREAD

**Issue by zheng17** [misha130/datepicker-ionic2#8]
Hi Misha130,
When I try to to build the project by using  the command 'ionic build android --prod', it give me error. But it works if I remove the '--prod' and that will build in dev mode.   I did some search on google, some guys said that is because your datepicker module does not support aot compiler.   

Please help. Here is the error details:

> ionic-app-scripts build "--prod"

[10:17:57]  ionic-app-scripts 1.0.0 
[10:17:57]  build prod started ... 
[10:17:57]  clean started ... 
[10:17:57]  clean finished in less than 1 ms 
[10:17:57]  copy started ... 
[10:17:57]  ngc started ... 
[10:17:58]  build prod failed: Unexpected value 'DatePickerModule' imported by the module 'AppModule' 
[10:17:58]  ionic-app-script task: "build" 
[10:17:58]  Error: Unexpected value 'DatePickerModule' imported by the module 'AppModule' 
Error: Unexpected value 'DatePickerModule' imported by the module 'AppModule'
    at /Users/johnlu/Documents/logile/hybrid/newsvn/ELM/node_modules/@angular/compiler/bundles/compiler.umd.js:14675:33
    at Array.forEach (native)
    at CompileMetadataResolver._loadNgModuleMetadata (/Users/johnlu/Documents/logile/hybrid/newsvn/ELM/node_modules/@angular/compiler/bundles/compiler.umd.js:14660:51)
    at CompileMetadataResolver.getUnloadedNgModuleMetadata (/Users/johnlu/Documents/logile/hybrid/newsvn/ELM/node_modules/@angular/compiler/bundles/compiler.umd.js:14636:23)
    at addNgModule (/Users/johnlu/Documents/logile/hybrid/newsvn/ELM/node_modules/@angular/compiler/bundles/compiler.umd.js:12944:43)
    at /Users/johnlu/Documents/logile/hybrid/newsvn/ELM/node_modules/@angular/compiler/bundles/compiler.umd.js:12957:16
    at Array.forEach (native)
    at _createNgModules (/Users/johnlu/Documents/logile/hybrid/newsvn/ELM/node_modules/@angular/compiler/bundles/compiler.umd.js:12956:28)
    at analyzeNgModules (/Users/johnlu/Documents/logile/hybrid/newsvn/ELM/node_modules/@angular/compiler/bundles/compiler.umd.js:12700:16)
    at analyzeAndValidateNgModules (/Users/johnlu/Documents/logile/hybrid/newsvn/ELM/node_modules/@angular/compiler/bundles/compiler.umd.js:12704:20)

npm ERR! Darwin 15.6.0

**Comment by misha130** [comment]
Thank you for opening the issue.
Its because AOT is currently making things complicated
I've been working on fixing this the last couple of days and to no avail. 
Hopefully I'll put a new version in a couple of days.

**Comment by ourarash** [comment]
I am experiencing the same issue:

```
> ionic-app-scripts build "--prod"

[13:53:43]  ionic-app-scripts 1.0.1
[13:53:43]  build prod started ...
[13:53:43]  preprocess started ...
[13:53:43]  preprocess finished in 2 ms
[13:53:43]  clean started ...
[13:53:43]  clean finished in less than 1 ms
[13:53:43]  copy started ...
[13:53:43]  ngc started ...
[13:53:46]  build prod failed: Unexpected value 'DatePickerModule' imported by the module 'AppModule'
[13:53:46]  ionic-app-script task: "build"
[13:53:46]  Error: Unexpected value 'DatePickerModule' imported by the module 'AppModule'
Error: Unexpected value 'DatePickerModule' imported by the module 'AppModule'

```

**Comment by misha130** [comment]
Fixed with version 2.2.11
